_posts/2020-08-20-COVID19-Illustrations-2020.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: COVID-19 ILLUSTRATIONS
+date: August 20, 2020
+categories:
+  - Covid19
+  - Pandemic
+  - open-source
+  - communities
+  - design
+
+authors:
+  - hqrshguptq
+---
+
+<img src="https://user-images.githubusercontent.com/54597242/90833938-b1f4c800-e366-11ea-95e3-07747a9aad15.png">
+
+Based on [recommendations from The World Health Organization](https://www.who.int/emergencies/diseases/novel-coronavirus-2019)  and others, you’ll find symbols to communicate good hygiene and social distancing. While we can’t be like front line brave medical professionals across the globe, we hope these icons/illustrations aid in communicating some of the most important things people can do to protect themselves and their communities.
+
+#### Free COVID-19 Awareness Icons
+
+All solid COVID-19 illustrations are available [Open Source](https://app.biorender.com/). A trial for some period to incorporate these can be availed at [bioRENDER](https://app.biorender.com/).
+Use them however you can in your signage, reports, sites, apps and projects. And please share any of your efforts – we’d love to see them and help boost awareness and helpful information.
+
+<img src="https://user-images.githubusercontent.com/54597242/90834044-f3857300-e366-11ea-9863-7c0392f04dbb.png">
+<img src="https://user-images.githubusercontent.com/54597242/90834072-05671600-e367-11ea-8fc8-edea9c3e69a9.png">
+
+## About BioRender
+<img src="https://user-images.githubusercontent.com/54597242/90834223-570fa080-e367-11ea-97a9-4a75fc7e0018.png" align='left' width='125px' height='125px'>[BioRender](https://app.biorender.com/) envision a world where science is universally communicated and understood through visuals. Represent science with intuitive visuals that can be understood by everyone. ACCESSIBLE. Anyone can use the platform - no drawing or design skills required.
```

Comment:
The `align='left'` part kind of messes up the layout a little bit (screenshot attached below). I would probably suggest `align='center'` ? Unless you really want it to be left aligned.

![image](https://user-images.githubusercontent.com/9530293/92017446-6df3c100-ed71-11ea-8600-efbb344fd187.png)

Reply:
Yeah,Obviously It could go like that.
However, What I did was to try to implement as screen-shotted below.
![Screenshot 2020-09-03 at 3 08 03 AM](https://user-images.githubusercontent.com/54597242/92039447-b1125c00-ed92-11ea-8d59-636024ef322d.png)

Reply:
I do understand that. If you absolutely want that way, you can add some inline styles to the `<img>` tag like:
```html
<img src="https://user-images.githubusercontent.com/54597242/90834223-570fa080-e367-11ea-97a9-4a75fc7e0018.png" style="margin: 0; margin-right: 16px;" width="125px" height="125px" align="left">
```

In that case, it would look like:
![image](https://user-images.githubusercontent.com/9530293/92083960-5efa2680-ede4-11ea-8580-15f00f4ea9de.png)

Reply:
Sure Man ! Looking Amazing 💯